libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/validator/VerifiableCredentialJwtValidator.java

```diff
@@ -58,37 +60,46 @@ public VerifiableCredentialJwtValidator(
         this.claimsVerifierFactory = claimsVerifierFactory;
     }
 
-    public void validate(
-            SignedJWT verifiableCredential, CriConfig credentialIssuerConfig, String userId)
+    public List<VerifiableCredential> validate(
+            List<String> vcs, String criId, ECKey signingKey, String componentId, String userId)
             throws VerifiableCredentialException {
-        LOGGER.info(LogHelper.buildLogMessage("Validating Verifiable Credential."));
-        ECKey signingKey;
-        try {
-            signingKey = credentialIssuerConfig.getParsedSigningKey();
-        } catch (ParseException e) {
-            LOGGER.error(
-                    LogHelper.buildErrorMessage("Error parsing credential issuer public JWK", e));
-            throw new VerifiableCredentialException(
-                    HTTPResponse.SC_SERVER_ERROR, ErrorResponse.FAILED_TO_PARSE_JWK);
+        var validVcs = new ArrayList<VerifiableCredential>();
+        for (String vc : vcs) {
+            validVcs.add(validate(vc, criId, signingKey, componentId, userId));
         }
-        validate(verifiableCredential, signingKey, credentialIssuerConfig.getComponentId(), userId);
-    }
 
-    public void validate(
-            SignedJWT verifiableCredential, ECKey signingKey, String componentId, String userId)
-            throws VerifiableCredentialException {
-        validateSignatureAndClaims(verifiableCredential, signingKey, componentId, userId);
-        validateCiCodes(verifiableCredential);
-        LOGGER.info(LogHelper.buildLogMessage("Verifiable Credential validated."));
+        return validVcs;
     }
 
-    public void validateSignatureAndClaims(
```

Comment:
This is removed because was only used for CiMit VCs and I've simplified the code since CiMit VCs could also be validated that their codes are valid

Reply:
I assume you've tested this(?) - Just making sure that it doesn't fall over when receiving the CIMIT VC format

Reply:
Annoyingly, this isn't allowed - I'll revert. The CiMit evidence is a different shape

Reply:
Ah, shame. We can always revisit in part 2, if we end up parsing into distinct classes at this point.

`if (vc instanceof IdentityCheckCredential) { ... }` or whatever